### PR TITLE
OLS-398: Token counter checks in e2e tests

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Token counter checks in e2e tests

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-398](https://issues.redhat.com//browse/OLS-398)
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Run `make test-e2e`

### CODE REVIEW

File: tests/e2e/test_api.py [hunk 1]
```diff
@@ -74,6 +74,34 @@ def get_model_provider_counter_value(
...
+            # strip suffix
+            return value[: value.find('"')]
+
+    # info node was not found
+    return None
+
+
+def get_model_and_provider(client):
+    """Read configured model and provider from metrics."""
+    """Retrieve counter value from metrics."""
```
Comment (onmete) [inline tests/e2e/test_api.py:95]: double docstring?

Comment (tisnik) [reply tests/e2e/test_api.py]: should be fixed now

**Pull request merged**